controller/ev/chargepoint/cp.py

```diff
@@ -93,22 +102,28 @@ def get_file():
 
 def save_to_file(load):
     global _DATAFILE
+    sg_load = load['sgLoad']
     if DEBUG:
         print(
-            str(time.time()) +
+            "time: " + str(time.time()) +
             "  total_load @ sgID(" +
             str(load["sgID"]) + ") = " +
-            load['sgLoad'] + "\n"
+            sg_load + "\n"
         )
-    datafile = get_file()
-    if DEBUG:
-        print("Writing to file " + str(datafile))
+    if (sg_load == '0.000'):
```

Comment:
Might want to convert to float first, in case the API eventually returns "0.00".

```suggestion
    try:
      sg = float(sg_load)
    except ValueError:
      sg = None
    if not sg:
```

Reply:
Was concerned with floating point crumbs and round off errors .. give ChargePoint returns a string, decided to stick with it.